Bottom: 426c83f9550d87d6eaa7d6b5e334d055e8c30517
Top:    b7195ee101f39b9e77326b39549e4e5dec704fa8
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 16:49:30 -0500

Use unprivileged ports for buildbot master server


---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 64802d00..fa19756e 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -2,16 +2,12 @@
 # defaults file for buildbot
 
 # The hostname passed as an envirnment variable into the container
-container_addr_map:
-  buildbot:
-    ip_addr: '127.0.0.1'
-    hostname: buildbot.example.com
-
-# the port mappings used when starting the container
-container_port_map:
-  buildbot:
-    port_args: 
-      - '{{ container_addr_map.buildbot.ip_addr }}:443:443'
+buildbot_ip_addr: '127.0.0.1'
+buildbot_hostname: buildbot.example.com
+buildbot_port_args: 
+  - '8080'
+  - '8443'
+  - '9989'
 
 # Network names
 buildbot_networks:
@@ -44,8 +40,8 @@ buildbot_volumes:
 
 buildbot_env:
   BUILDBOT_DATABASE: 'sqlite:///{{ buildbot_data_path }}/state.sqlite'
-  BUILDBOT_WEB_PORT: '80'
-  BUILDBOT_WEB_URL: 'http://{{ container_addr_map.buildbot.hostname }}/'
+  BUILDBOT_WEB_PORT: '8080'
+  BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
   BUILDBOT_SMTP_HOST: 'relay.dnsexit.com'
   BUILDBOT_SMTP_USER: 'Dean.cox'
   BUILDBOT_SMTP_PASS: 'no5156'
